src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,74 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { useMemo } from 'react';
+import { breadcrumbStyles, breadcrumbLinkStyles } from './Breadcrumb.style';
+import useTheme from 'hooks/useTheme';
+import BreadcrumbItem from './BreadcrumbItem/BreadcrumbItem';
+import isEmpty from 'lodash/isEmpty';
+import uniqueId from 'lodash/uniqueId';
+import { Link } from 'react-router-dom';
+import BreadcrumbCollapsed from './BreadcrumbCollapsed/BreadcrumbCollapsed';
+
+export type Props = {
+  separatorContent?: '*' | '>' | '/';
+  data: [];
+  breadcrumbItemClickHandler: () => void;
```

Comment:
what is this ?

Reply:
Initially was a clickHandler for a breadcrumb but as the implementation of the breadcrumb changed this became an obsolete feature. I will just remove it.